tests/pkg_cryptoauthlib_compare_sha256/Makefile

```diff
@@ -0,0 +1,11 @@
+include ../Makefile.tests_common
+
+# Test fails to build for these boards fails due to
+# redefinition of define AES_COUNT in library, which
+# is also defined in efm32 header files
+BOARD_BLACKLIST := stk3600 stk3700
```

Comment:
this could just be moved to the Make file.dep file of the package, and thus be removed from _both_ test applications (with the benefit of less code duplication)

Reply:
I explicitly requested to put it here. The need to blacklist these boards is stupid and simply related to name collisions of `#define`s from the efm32 vendor header and the library. Yet, we did not find an elegant solution to this problem. The intention is to make this "as visible as possible" to the user. And hopefully we find a better solution in future.